src/ValueFormatters/TimeFormatter.php

```diff
@@ -17,40 +15,37 @@
  * @author H. Snater < mediawiki@snater.com >
  */
 class TimeFormatter extends ValueFormatterBase {
+
+	/**
+	 * @deprecated since 0.7, use \ValueParsers\TimeParser::CALENDAR_GREGORIAN instead.
+	 */
 	const CALENDAR_GREGORIAN = 'http://www.wikidata.org/entity/Q1985727';
+
+	/**
+	 * @deprecated since 0.7, use \ValueParsers\TimeParser::CALENDAR_JULIAN instead.
+	 */
```

Comment:
What's the advantage of having this in the parser instead of the formatter? How about putting it in the value?